app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -91,6 +91,11 @@ export class SearchAndAddOrder extends React.Component {
     });
   }
 
+  closeFormsOnTabChange = () => {
+    this.props.selectDrugSuccess('');
```

Comment:
Why the use of an empty string as a parameter? Why not using a null value, i.e not sending anything at all?